### PR TITLE
Replacing deprecated sketch editor with new geometry editor

### PR DESCRIPTION
**Pull request by mfeigl**
### Description

 - The Sketch Editor is now deprecated and a the new feature Geometry Editor is replaced it; therefore, the `Sketch On Map` sample is removed, as the `Create And Edit Geometries` sample features the current workflow and advised feature usage.
 - The `Create And Save Kml File` sample has its sketch editor references replaced with the current geometry editor functionality.

Checklist:
- [x] Set target branch to v.next (next release)
- [x] Request a reviewer
- [x] Assign a reviewer
- [x] Tag reviewer in comment

### CODE REVIEW

**Comment by mfeigl** [comment]
@mbcoder could you please be the first reviewer of this PR?

**Review by mbcoder — APPROVED** [review]
@mfeigl this change looks good.

**Comment by mfeigl** [comment]
@jenmerritt could you please be the second reviewer of this PR?

File: kml/create-and-save-kml-file/src/main/java/com/esri/samples/create_and_save_kml_file/CreateAndSaveKMLFileController.java [hunk 1]
```diff
@@ -75,22 +75,22 @@ public void initialize() {
...
+    geometryTypeComboBox.getSelectionModel().selectedItemProperty().addListener(o -> startSketch());
 
     // start with POINT selected
-    map.addDoneLoadingListener(() -> sketchCreationModeComboBox.getSelectionModel().select(0));
+    map.addDoneLoadingListener(() -> geometryTypeComboBox.getSelectionModel().select(0));
 
-    // show style controls relevant to the selected sketch creation mode
-    colorPicker.visibleProperty().bind(sketchCreationModeComboBox.getSelectionModel().selectedItemProperty().isNotEqualTo(SketchCreationMode.POINT));
-    pointSymbolComboBox.visibleProperty().bind(sketchCreationModeComboBox.getSelectionModel().selectedItemProperty().isEqualTo(SketchCreationMode.POINT));
+    // show style controls relevant to the selected geometry creation mode
```
Comment (jenmerritt) [inline kml/create-and-save-kml-file/src/main/java/com/esri/samples/create_and_save_kml_file/CreateAndSaveKMLFileController.java]:
```suggestion
    // show style controls relevant to the selected geometry type
```

File: kml/create-and-save-kml-file/src/main/java/com/esri/samples/create_and_save_kml_file/CreateAndSaveKMLFileController.java [hunk 2]
```diff
@@ -144,9 +144,9 @@ private void handleKeyReleased(KeyEvent keyEvent) {
     if (keyEvent.getCode() == KeyCode.ESCAPE) {
       // clear the current sketch and start a new sketch
       startSketch();
-    } else if (keyEvent.getCode() == KeyCode.ENTER && sketchEditor.isSketchValid()) {
+    } else if (keyEvent.getCode() == KeyCode.ENTER) {
       // project the sketched geometry to WGS84 to comply with the KML standard
-      Geometry sketchGeometry = sketchEditor.getGeometry();
+      Geometry sketchGeometry = geometryEditor.getGeometry();
       Geometry projectedGeometry = GeometryEngine.project(sketchGeometry, SpatialReferences.getWgs84());
```
Comment (jenmerritt) [inline kml/create-and-save-kml-file/src/main/java/com/esri/samples/create_and_save_kml_file/CreateAndSaveKMLFileController.java]: Maybe one to think about/discuss - I'm not sure what's better but just jumped out. There is the use of the term "sketch" here. Is that still a term we want to use? Reading the doc my initial thought is to use `createGeometry()` in place of `startSketch()` and e.g. `geometry` in place of `sketchGeometry`.
There's also a few uses of "sketch" in the first couple of sections of the Readme. Maybe we don't mind but one to consider!

**Review by jenmerritt — COMMENTED** [review]
When this is merged into `main` as well, worth mentioning it to Steve as the website samples will also be auto updated so he can verify things there.

File: kml/create-and-save-kml-file/src/main/java/com/esri/samples/create_and_save_kml_file/CreateAndSaveKMLFileController.java (hunk 2 above)
Comment (mfeigl) [reply kml/create-and-save-kml-file/src/main/java/com/esri/samples/create_and_save_kml_file/CreateAndSaveKMLFileController.java]: Very good point, I was thinking about these as well. I agree with you these soun weird in the new context, changed them now.

**Comment by mfeigl** [comment]
Thanks @jenmerritt , I accepted your comments and ready for re-review

**Review by jenmerritt — APPROVED** [review]
LGTM @mfeigl 👍

**Comment by mfeigl** [comment]
Thank you @jenmerritt !

**Pull request merged**